docs/source/models/cause.rst

```diff
@@ -171,29 +174,321 @@ Common data sources for cause models
+  or to severe disease. In these cases, incidence rates specific to mild and
+  severe disease are needed to inform the specific transitions present in the
+  cause model.
+
+Lastly, see the section on `hazard rates`_ in non-standard data sources below
+to determine when hazard rates may be preferable to annual incidence rates as
+a data source for cause model transition rates.
+
+Remission
 ++++++++++
 
-Cause-specific mortality
+Remission rates are defined as the **number of newly recovered cases from a
+condition that occur per person-year of the population with the condition.**
+
+  For example, the remission rate of type II diabetes in the United States is
+  2.4 per 10,000 person-years.
+
+  This suggests that if we followed 10,000 individuals with type II diabetes
+  for one year each, we would expect to see 2.4 individuals recover from type
+  II diabetes.
+
+Remission rates can be used to estimate cause model transition rates and
+represent the probability that a simulant in an infected (with condition)
+state will transition to a non-infected (without condition) state.
+
+  For example, with a time step of one year, the probability that a simulant
+  in the infected state in a model of type II diabetes in the United States
+  will transition to a susceptible or recovered state within a timestep is
+  2.4*10^(-4), or 0.024%.
+
+.. note::
+
+  The considerations discussed in the incidence section above apply to
+  remission rates as well. See above_ for details.
+
+Disability Weights
+++++++++++++++++++
+
+Cause-Specific Mortality
 ++++++++++++++++++++++++
 
-Excess mortality
+Excess Mortality
 ++++++++++++++++
 
-Disability weight
-+++++++++++++++++
+Non-Standard Data Sources
+-------------------------
+
+Duration
+++++++++
+
+In certain situations, there may be restrictions on the amount of time a
+simulant may occupy a given cause model state. In these cases, it is important
+to specify the duration that simulants may remain in the state of interest.
+
+For example, in conditions that have acute and chronic phases, it may be
+necessary to specify the length of time an individual occupies the acute
+phase before transitioning into the chronic phase.
+
+  E.g. In a cause model of ischemic heart disease, a simulant may transition
+  from susceptible to a myocardial infarction state, where they remain for
+  28 days, before they transition to a ischemic heart disease state.
+
+Restrictions
+++++++++++++
+
+In addition to time-related restrictions discussed above, certain situations
+may require additional restrictions to be placed on cause model states.
+Examples of possible restrictions include:
+
+- Age range restrictions (e.g. only simulants under 5 years old may
+  enter this state)
+- Sex restrictions (e.g. only female simulants may enter this state)
+- Restrictions related to states in other cause models (e.g. only simulants
+  who are susceptible to condition X may enter this state)
+- Etc.
+
+Hazard Rates
+++++++++++++
```

Comment:
Ali to move to PR about survival analysis

Reply:
done